settings.json.default.json

```diff
@@ -42,7 +42,7 @@
         },
         "app": {
           "root": "/",
-          "host": "../..",
+          "host": "../",
```

Comment:
In a release, Fauxton should be at `/_utils/` so we have to get back one "folder".